### PR TITLE
Ch redesign about page 126584711

### PR DESCRIPTION
**Pull request by andela-cnwosu**
#### What does this PR do?
           About page redesign.
#### Description of Task to be completed?
          Make page responsive and add faq questions. Make data dynamic on the page.
#### How should this be manually tested?
          Clone repository and start the server. View page on localhost:3000/about
#### What are the relevant pivotal tracker stories?
          126584711
#### Screenshots (if appropriate)
screen shot 2016-08-10 at 11 56 04 am

### CODE REVIEW

File: app/controllers/welcome_controller.rb [hunk 1]
```diff
@@ -15,6 +15,13 @@ def popular
     renders(@events)
   end
 
+  def about
+    @attendees = Attendee.count
+    @tickets = UserTicket.count
```
Comment (SundayAdefila) [inline app/controllers/welcome_controller.rb]: `@attendees` and `@tickets` as variable names does not rightly reflect what these variables are, which are counts. Kindly check variable namings.

File: app/controllers/welcome_controller.rb [hunk 2]
```diff
@@ -15,6 +15,13 @@ def popular
     renders(@events)
   end
 
+  def about
```
Comment (andela-sadefila) [inline app/controllers/welcome_controller.rb]: `@attendees`, `@tickets` and `@sponsors` as variable names does not rightly reflect what these variables are, which are counts. Kindly check variable naming.

File: app/views/welcome/about.html.erb [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+            Card Title
+          </div>
+          <div class="feature-content">
+            <p>I am a very simple card. I am good at containing small bits of information.
+            I am convenient because I require little markup to use effectively.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
```
Comment (andela-sadefila) [inline app/views/welcome/about.html.erb]: Awesome line indentations! 👍

File: app/views/welcome/about.html.erb [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+    <div class="card">
+      <div class="card-image">
+        <img src="https://static.artofwhere.net/img/home-tiles/wholesale-device-cases.jpg">
+        <div class="feature-details">
+          <div class="feature-title">
+            Card Title
+          </div>
+          <div class="feature-content">
+            <p>I am a very simple card. I am good at containing small bits of information.
+            I am convenient because I require little markup to use effectively.</p>
```
Comment (andela-sadefila) [inline app/views/welcome/about.html.erb]: NITPICK: Remove/replace placeholder texts

File: app/views/welcome/about.html.erb [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+        <div class="collapsible-header">What does it cost to create an event?</div>
+        <div class="collapsible-body"><p>Lorem ipsum dolor sit amet.</p></div>
+      </li>
+      <li>
+        <div class="collapsible-header">Where are my tickets?</div>
+        <div class="collapsible-body"><p>Lorem ipsum dolor sit amet.</p></div>
+      </li>
+      <li>
+        <div class="collapsible-header">How to contact the event manager?</div>
+        <div class="collapsible-body"><p>Lorem ipsum dolor sit amet.</p></div>
```
Comment (andela-sadefila) [inline app/views/welcome/about.html.erb]: NITPICK: Remove/replace placeholder texts

**Comment by andela-sadefila** [comment]
Closing PR because of failing tests. Fix tests and reopen when ready.

**Pull request closed without merging**